tide_landing_page.install

```diff
@@ -1265,3 +1279,37 @@ function tide_landing_page_update_8032() {
   $field_config->set('description', $description);
   $field_config->save();
 }
+
+/**
+ * Add the new paragraph type tide_automated_listing.
+ */
+function tide_landing_page_update_8033() {
+  $result = \Drupal::service('module_installer')->install(['tide_automated_listing']);
+  if (!$result) {
+    throw new \Exception(sprintf('Unable to install module %s', 'tide_automated_listing'));
+  }
+
+  /** @var \Drupal\Core\Extension\ModuleHandlerInterface $moduleHandler */
+  $moduleHandler = \Drupal::service('module_handler');
+  $field_config = FieldConfig::loadByName('node', 'landing_page', 'field_landing_page_component');
+  if ($field_config) {
+    $handler_settings = $field_config->getSetting('handler_settings');
+    $dependencies_settings = $field_config->getSetting('dependencies');
+
+    // Add the Automated Listing paragraph to Landing Page component if exists.
+    if ($moduleHandler->moduleExists('tide_automated_listing')) {
+      $handler_settings['target_bundles']['automated_card_listing'] = 'automated_card_listing';
+      $handler_settings['target_bundles_drag_drop'][] = [
```

Comment:
@asarfara `target_bundles_drag_drop` is an assoc array with the bundle as its keys.